lighthouse-core/audits/sized-images.js

```diff
@@ -61,10 +84,12 @@ class SizedImages extends Audit {
     const unsizedImages = [];
 
     for (const image of images) {
-      const width = image.attributeWidth;
-      const height = image.attributeHeight;
+      const attrWidth = image.attributeWidth;
+      const attrHeight = image.attributeHeight;
+      const cssWidth = image.propertyWidth;
+      const cssHeight = image.propertyHeight;
       // images are considered sized if they have defined & valid values
-      if (!width || !height || !SizedImages.isValid(width) || !SizedImages.isValid(height)) {
+      if (SizedImages.unsizedImage(attrWidth, attrHeight, cssWidth, cssHeight)) {
```

Comment:
maybe just pass the `image` to this function?

Reply:
yeah that makes sense, specially if the function is `isUnsizedImage`, I'll do that!